### PR TITLE
Updated the versions for some dependencies.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
@bshaffer
Can you take a quick look?

This CL will fix the broken build on travis of this project.
The motivation here is to make a pull request to the original repo[1].

[1]: https://github.com/satooshi/php-coveralls

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2013 Kitamura Satoshi
+Copyright (c) 2015 Google Inc
```
Comment (bshaffer) [inline LICENSE]: I don't think this will fly with the coverall guys :-P

**Comment by bshaffer** [comment]
You're saying we'll depend on this version of coveralls rather than `satooshi/php-coveralls`? And add your fork to packagist?

File: LICENSE (hunk 1 above)
Comment (tmatsuo) [inline LICENSE]: This is a requirement from our OSS team

**Comment by tmatsuo** [comment]
No I will create a new pull request to satooshi's repo after this is merged here.

**Comment by bshaffer** [comment]
There's no way @satooshi is going to allow a `Google` copyright in his license file...

**Comment by bshaffer** [comment]
also I'm not sure we are even allowed to do that... And I don't understand why that would be a legal requirement anyway, as long as the library has the correct license. Very confusing.

**Comment by tmatsuo** [comment]
This is not for an intention of overtaking the project. Just put Google's name alongside of the original author because we're actually contributing to the project. I don't see any issues here.

**Comment by tmatsuo** [comment]
Ok, I think I misunderstand the OSS team's policy. Removed the copyright notice.

**Comment by tmatsuo** [comment]
It turned out we need to at least add AUTHORS file. PTAL @bshaffer

**Comment by bshaffer** [comment]
LGTM

**Comment by tmatsuo** [comment]
Thanks!

**Pull request merged**